packages.ini

```diff
@@ -354,13 +380,16 @@ brew_requires = libmaxminddb
 
 [packaging==21.3]
 
+[paramiko==2.8.0]
```

Comment:
looks like this needs to be at least 2.9.3 to be fixed: https://github.com/paramiko/paramiko/commit/11232cca7be7bc5fc51eef1e8869db4450bc79e3